### PR TITLE
Add installation guide and improve migration and model example in README.md

### PR DESCRIPTION
**Pull request by MetalMatze**
Hey @phaza,

can you add to the migration and model examples how to use the other geometry types? That would be great.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,43 +14,66 @@ Laravel postgis extension
...
+
+    'Phaza\LaravelPostgis\DatabaseServiceProvider',
+
+That's all.
+
 ## Usage
 
-First of all, enable postgis (See note further down)
+A method called enablePostgis() (and disablePostgis()) is included in the Blueprint object.  
+They work on newer postgres installations, but I recommend enabling postgis manually for now.
```
Comment (phaza) [inline README.md]: The problem, as I realised not too long ago, is that you need superuser privileges to enable postgis.
Many/Most? don't use their database superuser as application user, and hence the enablePostgis() call will fail.

I think we should actually just recommend that people enable Postgis with `CREATE EXTENSION postgis` as a database superuser.

File: README.md [hunk 2]
```diff
@@ -14,43 +14,66 @@ Laravel postgis extension
...
+If you don't want a model and just a migration run
+
+    php artisan make:migration create_locations_table
+
+Open the created migrations with your editor.
+
 ```PHP
+use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Database\Migrations\Migration;
-use Phaza\LaravelPostgis\Schema\Blueprint;
```
Comment (phaza) [inline README.md]: It's actually better to typehint with "our" Blueprint, as that will give proper code completion in IDE's like PHPStorm and Sublime text for our custom column types.

Comment (MetalMatze) [inline README.md]: Alright, I'm gonne add that back.

File: README.md (hunk 1 above)
Comment (MetalMatze) [inline README.md]: Sounds good to me. Using those methods in my migrations didn't work as excepted with up and down.

**Comment by MetalMatze** [comment]
I've updated the README.md.

**Pull request merged**